CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 #### VM Internals
 - Removed unused `find_lone_leaf()` function from the Advice Provider (#1262).
+- Changed fields type of the `StackOutputs` struct from `Vec<u64>` to `Vec<Felt>` (#1268).
```

Comment:
This is a breaking change (i.e., it changes the public interface). We should mark it as such.

Reply:
Yep - that's fine.